src/test/ui/attributes/key-value-expansion.stderr

```diff
@@ -18,7 +18,7 @@ LL | bug!();
 error: unexpected expression: `{
                let res =
                    ::alloc::fmt::format(::core::fmt::Arguments::new_v1(&[""],
-                           &[::core::fmt::ArgumentV1::new_display(&"u8")]));
+                           &[::core::fmt::ArgumentV1::new_display(&"/*\u{ab}*/ u8 /*\u{bb}*/")]));
```

Comment:
Where do those comments inside the string come from? Is it how invisible delimiters are stringified?

Reply:
It was added in #96682, which landed recently. It's been extremely useful, the patches in this PR would have been twice as hard without it, not knowing where invisible delimiters are showing up.